DOCKER.md

```diff
@@ -40,3 +59,15 @@ Then, set the `DEBUG` environment variable and expose the port where to connect
 ```bash
 docker run --rm -e DEBUG=true -p 5005:5005 -v $PWD/test:/workspace jenkinsfile-runner:my-production-jenkins
 ```
+
+## Custom War Packager
+There is an alternative to Vanilla Dockerfile that gives a huge versatility when it comes to generate the docker images.
```

Comment:
The flow also generates an off-Docker executable though we have never tested it. Better to keep it as is